sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: ws-qualityprofiles.proto
+
+package org.sonarqube.ws;
+
+public final class QualityProfiles {
+  private QualityProfiles() {}
+  public static void registerAllExtensions(
+      com.google.protobuf.ExtensionRegistry registry) {
+  }
+  public interface WsSearchResponseOrBuilder extends
+      // @@protoc_insertion_point(interface_extends:sonarqube.ws.rules.WsSearchResponse)
+      com.google.protobuf.MessageOrBuilder {
+
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    java.util.List<org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile> 
+        getProfilesList();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile getProfiles(int index);
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    int getProfilesCount();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    java.util.List<? extends org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfileOrBuilder> 
+        getProfilesOrBuilderList();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfileOrBuilder getProfilesOrBuilder(
+        int index);
+  }
+  /**
+   * Protobuf type {@code sonarqube.ws.rules.WsSearchResponse}
+   *
+   * <pre>
+   * WS api/qualityprofiles/search
+   * </pre>
+   */
+  public static final class WsSearchResponse extends
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2160)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this class is 465 which is greater than 200 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassCyclomaticComplexity)